packages/create-cloudflare/src/workers.ts

```diff
@@ -42,13 +42,16 @@ export const runWorkersGenerator = async (args: C3Args) => {
 		args,
 		originalCWD,
 	};
+	const { preexisting } = await getTemplate(ctx);
 
-	ctx.args.ts = await processArgument<boolean>(ctx.args, "ts", {
```

Comment:
I think we actually want the following:

```	
  const { preexisting } = await getTemplate(ctx);

	if (preexisting) {
		ctx.args.ts = false;
	}

	await processArgument<boolean>(ctx.args, "ts", {
		type: "confirm",
		question: "Do you want to use TypeScript?",
		label: "typescript",
		defaultValue: C3_DEFAULTS.ts,
	});
```

`processArgument` won't prompt the user for input if the first param passed to it is already defined. So a pattern we use in other places where we want to infer some arguments from others is to directly set the inferred value on `ctx.args`. 

In addition, `processArgument` returns a value but also has the important side-effect of informing the user about the selection (note the typescript question with the proposed changed):

![image](https://github.com/cloudflare/workers-sdk/assets/204386/e9dc4cfc-84a1-4954-976e-4b2089ac566b)

Reply:
Also FYI we're [using this pattern ](https://github.com/cloudflare/workers-sdk/blob/main/packages/create-cloudflare/src/cli.ts#L91)to infer the `pre-existing` type when `--existing-script foo` is used.